tests/unit/test_adapter_registry.py

```diff
@@ -21,12 +20,3 @@ def test_adapter_registry_unknown_model():
 
     with pytest.raises(KeyError, match="Unknown model 'unknown'"):
         load_adapter("unknown")
-
-
-def test_adapter_registry_import_error():
```

Comment:
We will never have the possibility to test this as we include all dependencies already when doing `pip install .[tests]`. However, do we actually need to?

Reply:
@swillner that's what the mocking is for, you can mock things so that they do whatever you want (in this case raise `ImportError` even though the module is actually installed). Can you put the test back in please?

Reply:
Done